rpcutils/proof.go

```diff
@@ -0,0 +1,128 @@
+package rpcutils
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/taproot-assets/proof"
+	mboxrpc "github.com/lightninglabs/taproot-assets/taprpc/authmailboxrpc"
```

Comment:
I think file name here should be more specific to `authmailboxrpc`. IMO `proof.go` is too general.

Reply:
Moved things around now, the (un)marshal function is now in the proof package.